script/ide/System/Database/Collection.lua

```diff
@@ -61,6 +62,11 @@ function Collection:GetParent()
 	return self.parent;
 end
 
+function Collection:GetProvider()
```

Comment:
Is this provider or just provider name? 

the function name is a bit confusing